src/Raven.Server/Web/Studio/LicenseHandler.cs

```diff
@@ -204,6 +207,74 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                 }
             }
         }
+
+        [RavenAction("/studio/whats-new", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
+        public async Task GetWhatsNewData()
+        {
+            if (ServerVersion.Build == ServerVersion.DevBuildNumber)
+            {
+                HttpContext.Response.StatusCode = (int)HttpStatusCode.BadRequest;
+                return;
+            }
+
+            var license = ServerStore.LoadLicense();
+
+            if (license == null)
+            {
+                HttpContext.Response.StatusCode = (int)HttpStatusCode.BadRequest;
+                return;
+            }
+            
+            var licenseId = license.Id.ToString();
+            
+            var request = new AboutViewRequest() { CurrentFullVersion = ServerVersion.FullVersion, LicenseId = licenseId };
+
+            var requestPayload = JsonConvert.SerializeObject(request);
+
+            var content = new StringContent(requestPayload, Encoding.UTF8, "application/json");
+
+            var upgradeInfoResponse = await ApiHttpClient.Instance.PostAsync("api/v1/upgrade/get-upgrade-info", content);
+
+            if (upgradeInfoResponse.IsSuccessStatusCode == false)
+            {
+                HttpContext.Response.StatusCode = (int)upgradeInfoResponse.StatusCode;
```

Comment:
What about the response of an unsuccessful request? It might contain the error message. It might be useful.

Reply:
Described below